layers.qmd

```diff
@@ -265,7 +261,7 @@ ggplot(mpg, aes(x = displ, y = hwy)) +
   geom_smooth()
 ```
 
-Every geom function in ggplot2 takes a `mapping` argument.
+Every geom function in ggplot2 takes a `mapping` argument, which it inherits from `ggplot()`.
```

Comment:
I don't really get what that means. Inherits in what sense?

Reply:
Referring to global mapping vs. local mapping, I'll clarify